tests/functional/workloads/ocp/registry/test_registry_reboot_node.py

```diff
@@ -87,7 +87,7 @@ def test_registry_reboot_node(self, node_type, nodes):
         )(wait_for_nodes_status)(timeout=900)
 
         # Validate cluster health ok and all pods are running
-        self.sanity_helpers.health_check(tries=40)
+        self.sanity_helpers.health_check(tries=70)
```

Comment:
how did you come up with this new value?

Reply:
I have tested in loop and finally fixed this values for script to wait to get ceph Health OK